README.md

```diff
@@ -156,21 +178,25 @@ eval $(ssh-agent)
 ssh-add
 ```
 
-The environment variable `RIOTBASE` must be set to *absolute path* of the
-version of RIOT under test. E.g.
+#### Local Requirements
 
-```
-export RIOTBASE=$(readlink -f ../RIOT)
-```
+To be able to run tests locally the following requirements must be fulfilled:
 
-Some tests on the `native` platform need a certain number of TAP interfaces in a
-bridge or otherwise will be skipped. The most number of TAP interfaces to date
-is required for 3.5 "ICMPv6 stress test on native (neighbor cache
-stress)" (11 TAP interfaces) so to not skip that, all of them should be bridged.
+- udev rules mapping every `BOARD` to a specific `PORT`, e.g.: `/dev/riot-tty${BOARD}`.
+- a `makefile` included in `RIOT_MAKEFILES_GLOBAL_PRE` that sets `DEBUG_ADAPTER_ID`
+  and `PORT` for each `BOARD`.
 
-```sh
-sudo ${RIOTBASE}/dist/tools/tapsetup/tapsetup -c 11
-```
+See [multiple boards udev](http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev) for a detailed walkthrough.
+
+##### Limitations
+
+- Currently only tests that run on different `BOARD`'s will work out of the box,
+  it would currently require setting an additional variable to identify each
+  `BOARD`, see [multiple boards udev](http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev) for more detailes.
```

Comment:
You could move the line I mentioned above lower and just use

```suggestion
  `BOARD`, see [multiple boards udev] for more detailes.
```

here as well.